modify-vm-resources/modify-vm-resources.sw.yaml

```diff
@@ -42,6 +44,37 @@ functions:
       .vm.spec.template.spec.domain.cpu.threads else .vm_new_cpu_threads
       end
       }'
+  - name: successResult
+    type: expression
+    operation: '{
+        "completedWith":"success",
+        "message": "VM" + .vm_name + " in namespace " + .vm_namespace + " updated.",
+        "outputs":[
+          {
+              "key":"Console URL",
+              "value": $SECRET.cluster_console_url + "/k8s/ns/" + .vm_namespace + "/kubevirt.io~v1~VirtualMachine/" + .vm_name + "/console/standalone",
+              "format":"link"
+          }
+        ]
+      }'
+  - name: errorVMNotRunningResult
+    type: expression
+    operation: '{
+        "completedWith":"error",
+        "message": "VM" + .vm_name + " in namespace " + .vm_namespace + " not ready after the update after checking" +.vmStatusRunningRetries+" times: " + .vm.status
```

Comment:
to match the spacing  pls s/`+.vmStatusRunningRetries+"`/`+ .vmStatusRunningRetries +"`

Reply:
done